src/queries/vector-queries/sparse-vector-query.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+const { Query } = require('../../core');
+
+/**
+ * The sparse vector query executes a query consisting of sparse vectors, such as built by a learned sparse retrieval model
+ *
```

Comment:
suggestion: Could you please call out the specific version of Elasticsearch this is available from?

Reply:
good one. Also added specific versions to some of the parameters as they have been added later.